packages/shared/src/components/feeds/FeedNav.tsx

```diff
@@ -78,26 +79,34 @@ function FeedNav(): ReactElement {
   const scrollClassName = useScrollTopClassName({ enabled: !!featureTheme });
   const { feeds } = useFeeds();
   const { showPlusSubscription } = usePlusSubscription();
+  const { isCustomDefaultFeed, defaultFeedId } = useCustomDefaultFeed();
   const { openModal } = useLazyModal();
 
   const isHiddenOnboardingChecklistView =
     onboardingChecklistView === ChecklistViewState.Hidden;
 
   const urlToTab: Record<string, FeedNavTab> = useMemo(() => {
     const customFeeds = feeds?.edges?.reduce((acc, { node: feed }) => {
-      const feedPath = `${webappUrl}feeds/${feed.id}`;
       const isEditingFeed =
         router.query.slugOrId === feed.id && router.pathname.endsWith('/edit');
+      let feedPath = `${webappUrl}feeds/${feed.id}`;
+
+      if (!isEditingFeed && isCustomDefaultFeed && feed.id === defaultFeedId) {
+        feedPath = `${webappUrl}`;
+      }
+
       const urlPath = `${feedPath}${isEditingFeed ? '/edit' : ''}`;
 
       acc[urlPath] = feed.flags?.name || `Feed ${feed.id}`;
 
       return acc;
     }, {});
 
+    const forYouTab = isCustomDefaultFeed ? `${webappUrl}my-feed` : webappUrl;
```

Comment:
This also works for extension?

Reply:
@rebelchris This is for mobile/tablet, is it supposed to have special extension functionality? All the other buttons that don't redirect to webapp on desktop (following, explore etc) also redirect to webapp when clicked on FeedNav in mobile/tablet

Reply:
yeah we don't have to worry about mobile/tablet in extension